go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/tendermint/tendermint
+module github.com/cometbft/cometbft
```

Comment:
👍 

When this PR lands in a release, we should call out in release notes that consumers of this library (i.e. celestia-app and celestia-node) need to change the `replace` statement in go.mod